src/@batch-flask/ui/select/select-dropdown/select-dropdown.component.ts

```diff
@@ -124,4 +128,13 @@ export class SelectDropdownComponent {
         this.keyNavigator.items = this.rows;
         this.changeDetector.markForCheck();
     }
+
+    private _announceItems() {
```

Comment:
It would be better to localize these strings. [Here's an example](https://github.com/Azure/BatchExplorer/blob/765a876fcda26eeab06390b8a0be67a7a82a6ef6/src/app/components/pool/action/edit/edit-app-package/edit-app-package-form.component.ts#L64) of how to localize parametrized strings.